### PR TITLE
Ex1_blink_jbuican

### PR DESCRIPTION
**Pull request by jbuican14**
- create a simple blink plug-in 1

### CODE REVIEW

**Comment by jbuican14** [comment]
When I use http://www.jslint.com/ but it did not catch it. So I must have failed to double check that I missed the semicolon.

**Comment by afeld** [comment]
Yep, this is using [JSHint](http://jshint.com/), not JSLint.

File: blink.js [hunk 1]
```diff
@@ -1 +1,25 @@
...
+        var setTimer = setInterval(function() {
+          if (_this.style.visibility == 'visible') {
+            _this.style.visibility = 'hidden';
+          } else {
+            _this.style.visibility = 'visible';
+          }
+        }, delay);
+      } else {
+      }
+    });
```
Comment (jbuican14) [inline blink.js:23]: I added the ; that I missed it for the first pull request

**Comment by jbuican14** [comment]
Thank you. Do I pull to the right branch? I noticed that the url i pull to is : 
https://github.com/advanced-js/blink/pull/42
but not to the branch that i have as https://github.com/jbuican14/  -- this is all new to me and I don't really understand it much.

**Comment by afeld** [comment]
> Do I pull to the right branch?

I think you mean "push"? Anyway, the answer is yes!

**Comment by afeld** [comment]
Looks good, thanks!

**Pull request closed without merging**